phys/module_diag_solar.F

```diff
@@ -88,6 +90,9 @@ SUBROUTINE solar_diag (rho, dz8w, ph, phb, cldfrac3d, coszen, swdnb, swdnt, &
                clrnidx(i, j) = MISSING
              end if
 
+             !!! ACCUMULATED GHI
+             ghi_accum(i, j) = ghi_accum(i, j) + swdnb(i, j) * dt
```

Comment:
@twjuliano Any idea how many days of simulation can you use this formulation for without losing accuracy when running the model with single precision?

Reply:
@weiwangncar A one day test run leads to an accumulated GHI of ~3e7 J/m2. Given the limitations of single precision in Fortran, I estimate that we will begin to lose accuracy after about 30 days of simulation time.